src/usr/share/gpgmailer/gpgmailer.py

```diff
@@ -97,7 +96,7 @@ def start_monitoring(self):
                     encrypted_message = self._build_encrypted_message(
                         message_dict, loop_start_time)
 
-                    self.mailsender.sendmail(message_string=encrypted_message,
+                    self._sendmail(message_object=encrypted_message,
```

Comment:
Maybe call this mime_message or something? I don't like message_object. Of course it is an object!